src/extension.ts

```diff
@@ -97,7 +99,7 @@ function activate (context: vscode.ExtensionContext): Promise<void> {
 
 		// register code action providers
 		// Disable for now - https://github.com/appcelerator/vscode-appcelerator-titanium/issues/162
-		// vscode.languages.registerCodeActionsProvider({ scheme: 'file', pattern: viewFilePattern }, new ViewCodeActionProvider()),
+		vscode.languages.registerCodeActionsProvider({ scheme: 'file', pattern: viewFilePattern }, new ViewCodeActionProvider()),
```

Comment:
Should we move this into the `definitionProviderHelper.activate` call? If we don't, please remove the comment on line 101.